coalib/parsing/filters/FilterBase.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC, abstractmethod
+
+
+def overrides(interface_class):
+    def overrider(method):
+        assert(method.__name__ in dir(interface_class))
+        return method
+    return overrider
```

Comment:
Its not about `ABC` and `@abstractmethod` that @Makman2 is discussing about.
Its mainly custom `@overrides` decorator.